src/rTerminal.ts

```diff
@@ -189,51 +189,36 @@ export async function chooseTerminal(): Promise<vscode.Terminal | undefined> {
     for (let i = 0; i < vscode.window.terminals.length; i++){
         msg += `Terminal ${i}: ${vscode.window.terminals[i].name} `;
     }
-    if (vscode.window.terminals.length > 0) {
-        const rTermNameOptions = ['R', 'R Interactive'];
-        if (vscode.window.activeTerminal !== undefined) {
-            const activeTerminalName = vscode.window.activeTerminal.name;
-            if (rTermNameOptions.includes(activeTerminalName)) {
-                return vscode.window.activeTerminal;
-            }
-            for (let i = vscode.window.terminals.length - 1; i >= 0; i--){
-                const terminal = vscode.window.terminals[i];
-                const terminalName = terminal.name;
-                if (rTermNameOptions.includes(terminalName)) {
-                    terminal.show(true);
-                    return terminal;
-                }
-            }
-        } else {
-            msg += `B. There are ${vscode.window.terminals.length} terminals: `;
-            for (let i = 0; i < vscode.window.terminals.length; i++){
-                msg += `Terminal ${i}: ${vscode.window.terminals[i].name} `;
-            }
-            // Creating a terminal when there aren't any already does not seem to set activeTerminal
-            if (vscode.window.terminals.length === 1) {
-                const activeTerminalName = vscode.window.terminals[0].name;
-                if (rTermNameOptions.includes(activeTerminalName)) {
-                    return vscode.window.terminals[0];
-                }
-            } else {
-                msg += `C. There are ${vscode.window.terminals.length} terminals: `;
-                for (let i = 0; i < vscode.window.terminals.length; i++){
-                    msg += `Terminal ${i}: ${vscode.window.terminals[i].name} `;
-                }
-                console.info(msg);
-                void vscode.window.showErrorMessage('Error identifying terminal! Please run command "Developer: Toggle Developer Tools", find the message starting with "[chooseTerminal]", and copy the message to https://github.com/REditorSupport/vscode-R/issues');
 
-                return undefined;
-            }
-        }
-    }
+    const rTermNameOptions = ['R', 'R Interactive'];
+
+    // General identifier for terminals to ignore
```

Comment:
Can you add a comment saying why the string is `'Deactivate`'? (Like, one line mentioning that the Python extension creates hidden terminals including that string.)